designs/developer/add-45/sections/pipelines/pipelines.md

```diff
@@ -0,0 +1,83 @@
+---
+parent: Developer
+version: 4.5
+---
+
+# Pipelines section in Add Flows (added to 4.5)
+* Updates to the Pipelines section of the 'Add' page include moving the Pipelines section below the Resources section. In addition, the ‘Show pipeline visualization’ component will remain open even if the user selects a new builder image or resource during their session.
```

Comment:
Can we add back in the intro that was in the 4.3 documentation? 

The Pipelines section allows the user to enable a pipeline template for their application, component or service.
- A Pipelines section appears in +Add flows (for Import from Git, Developer Catalog, and From Dockerfile) when the Pipelines operator is installed.
- When the Pipelines operator is not installed, this section is not displayed

I would put this under the title like it is in 4.3.

Reply:
Yup I can add that info back to the intro.

Reply:
@lwrigh feel free to resolve these when you've tackled them!